internal/logging/error_codes.go

```diff
@@ -1,14 +1,16 @@
-package logging
-
-const S3_PATH_PARSE_ERROR = -10
-
-const COULD_NOT_FIND_CREDENTIALS = -20
-
-const PIPE_FAILURE = -30
-
-const DB_SCHEMA_SCAN_FAILURE = -40
-const DB_TRUNCATE_ERROR = -41
-const DB_DROP_ERROR = -42
-
-const DB_MISSING_TABLES = -50
-const PROTECTED_DATABASE = -51
+package logging

```

Comment:
Are there CR/LF, LF things going on here? Just asking. We try and do LF only in the repo; I ask, because I'm not sure what the changes were on every line.

Reply:
yea, it was CRLF, and my settings set it back to LF

Reply:
Groovy. TY.